app/components/Quest/WalletMenu.tsx

```diff
@@ -0,0 +1,47 @@
+import WalletButton from "../Buttons/ActionButton";
+import { useAccount, useConnectors } from "@starknet-react/core";
+import { useEffect } from "react";
+
+interface Props {
+    action: Function
+}
+
+export default function WalletMenu({ action }: Props) {
```

Comment:
Note for later: I also created a popup with the new style guide and Allan is currently working to improve it.
When ready, we will have to use the same popup everywhere.

Reply:
agree